packages/core/parcel-bundler/src/Bundle.js

```diff
@@ -274,10 +274,11 @@ class Bundle {
 
   getHash() {
     let hash = crypto.createHash('md5');
-    for (let asset of this.assets) {
+    let visit = asset => {
       hash.update(asset.hash);
-    }
-
+      Array.from(asset.depAssets.values()).forEach(visit);
```

Comment:
I think this will be a problem if there are circular dependencies.

Reply:
Thanks, I've added a test for that and a guard to prevent infinite recursion if there's a cycle.